scripts/observability/cli.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+#######################################
+# Install the SUSE Observability CLI
+#######################################
+observability_install_cli() {
+  curl -o- https://dl.stackstate.com/stackstate-cli/install.sh | STS_CLI_LOCATION=/usr/local/bin bash
```

Comment:
Would it be possible to check before that it's not installed already? (make the function idempotent)
Is there a way to for the version?